packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -38,6 +39,7 @@ export function generateIsomorphicTests(
     props: { [name: string]: any },
     children: { [name: string]: React.ReactNode },
     skipList: string[] = [],
+    classNameChildList: string[] = [],
```

Comment:
Need to document this new param in the function comment above.

Reply:
Done!